flask_openapi3/models/__init__.py

```diff
@@ -63,6 +64,7 @@ class APISpec(BaseModel):
     security: Optional[List[SecurityRequirement]] = None
     tags: Optional[List[Tag]] = None
     externalDocs: Optional[ExternalDocumentation] = None
+    webhooks: Optional[Dict[str, Union[Webhook, Reference]]] = None
```

Comment:
There's a mistake here.
yours: `webhooks: Optional[Dict[str, Union[Dict[str, PathItem], Reference]]] = None`
should be: `webhooks: Optional[Dict[str, Union[PathItem, Reference]]] = None`

Here is no need to create a new `webhook.py` file.

Reply:
Correct. Why I changed it to draft. I'll use a project as a test case and then request review again.